frontend/src/components/mail-verification/verification-failed/failed.tsx

```diff
@@ -0,0 +1,71 @@
+import { FC, useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+
+import Logo from '@assets/images/logo.svg';
+import { AppRoute } from '@common/enums/app/app';
+import { ButtonFill } from '@components/common/button-fill/button-fill';
+import { InputField } from '@components/common/input-field/input-field';
+import Container from '@mui/material/Container';
+import { emailApi } from '@services/request-verification-link/email-link-service';
+
+import styles from './styles.module.scss';
+import { emailSchema } from './validation-schema';
+
+const Failed: FC = (): React.ReactElement => {
+  const [email, setEmail] = useState({ email: '' });
+  const [validationError, setValidationError] = useState('');
+  const [requestLink] = emailApi.useRequestLinkMutation();
+  const navigate = useNavigate();
+
+  const onchangeHandler = async (
+    event: React.ChangeEvent<HTMLTextAreaElement>,
+  ): Promise<void> => {
+    setEmail({
+      email: event.target.value,
+    });
+  };
```

Comment:
`camelCase` naming should be preserved. Therefore you should use the `onChangeHandler`. Although, why do you use the object for the email? :thinking: